### PR TITLE
Fix #68: Changes entity instantiation class to reflection instantiate without constructor method.

### LINKED ISSUE #68: Errors with contructor method attributes when use entityNamespace.
When I use the `$mapper->entityNamespace = '\\MyApplication\\Entities';` in entities with constructor method attributes is generated php errors. You have a solution for this?

### PR DESCRIPTION
**Pull request by jackmakiyama**

### CODE REVIEW

File: library/Respect/Relational/Mapper.php [hunk 1]
```diff
@@ -426,31 +445,33 @@ protected function fetchSingle(
...
+
     protected function getNewEntityByName($entityName)
     {
         $entityName = $this->getStyle()->styledName($entityName);
         $entityClass = $this->entityNamespace . $entityName;
         $entityClass = class_exists($entityClass) ? $entityClass : '\stdClass';
-        return new $entityClass;
+        $entityReflection = new ReflectionClass($entityClass);
+
+        return $entityReflection->newInstanceWithoutConstructor();
```
Comment (henriquemoody) [inline library/Respect/Relational/Mapper.php]: The build is failing because `newInstanceWithoutConstructor()` is not available on PHP 5.3... :cry:

Comment (jackmakiyama) [inline library/Respect/Relational/Mapper.php]: :cry:

Comment (henriquemoody) [inline library/Respect/Relational/Mapper.php]: You can drop support for 5.3, them I will release a MINOR version.

Comment (jackmakiyama) [inline library/Respect/Relational/Mapper.php]: Done! =D

**Comment by felipecwb** [comment]
:+1:

**Comment by henriquemoody** [comment]
Version 0.7.0 has been created.

Thanks guys!

**Comment by henriquemoody** [comment]
PS.: I'm sorry by my delay: very busy weeks...

**Comment by jackmakiyama** [comment]
Thanks guys!
*----*

**Pull request merged**